Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Tool.Strip.Items/Controls/Advanced/ExpandedItem.cs

```diff
@@ -0,0 +1,30 @@
+﻿#region BSD License
+/*
+ * Use of this source code is governed by a BSD-style
+ * license or other governing licenses that can be found in the LICENSE.md file or at
+ * https://raw.githubusercontent.com/Krypton-Suite/Extended-Toolkit/master/LICENSE
+ */
+#endregion
+
+using Krypton.Toolkit.Suite.Extended.Tool.Strip.Items.Properties;
```

Comment:
Why are these here, ans in that order ?

Reply:
Been fixed, now appears in the `GlobalDeclarations` file